sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -666,11 +703,65 @@ const StartHTLCAssetLockListener = (
     assetId: string,
     recipientECert: string,
     lockerECert: string,
-    lockCallback: (c: Contract, d: string, t: string, i: string, r: string, l: string, v: string) => any,
+    lockCallback: (c: Contract, d: string, t: string, i: string, r: string, l: string, v: Hash, timeout: number) => any,
 ): void => {
     StartHTLCEventListener(contract, 'LockAsset', contractId, assetType, assetId, -1, recipientECert, lockerECert, lockCallback);
 }
 
+//TODO: Timed counterpart for timed AssetLockListener for reversion
+const assetLockCallbackTimed = (
+    contract: Contract, 
+    contractID: string, 
+    assetType: string, 
+    assetID: string, 
+    recipientECert: string, 
+    lockerECert: string, 
+    hash: Hash,
+    expiryTime: number
+): void => {
+    //Compare expiryTimeSec with currentTimeSec, which is number of milliseconds since epoch (L174)
+    const currTimeSecs = Math.floor(Date.now()/1000);
+
+    const reclaimCallback = async function(contract: Contract, assetType: string, assetID: string, recipientECert: string, contractID: string) {
+        //TODO: Call isAssetLockedInHTLC -> Hasn't been claimed yet. If true, do nothing. If false, either do the following cases:
+        const [islocked, isLockedQueryError] = await helpers.handlePromise(isAssetLockedInHTLCqueryUsingContractId(contract,contractID));
```

Comment:
Insert space after `contract,`. In general, have a space between a comma and the next parameter in a function call. (Aesthetics.)